app/scripts/ui.js

```diff
@@ -101,7 +156,7 @@ async function queryCurrentActiveTab(windowType) {
   });
 }
 
-function initializeUi(activeTab, container, connectionStream, cb) {
+function initializeUi(activeTab, connectionStream, cb) {
```

Comment:
looks like `container` is still used on line 169?

Reply:
`container` is now moved [here](https://github.com/MetaMask/metamask-extension/pull/14781/files#diff-5a9d964db9d7c2fe207e21470884b2710e54723e0c1e6e6f7fd264706604fd6cR27).